src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateAggregate.tt

```diff
@@ -0,0 +1,134 @@
+﻿<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.Linq" #>
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Data.Sqlite
+{
+    partial class SqliteConnection
+    {
+<#
+    var ordinal = new[] { "first", "second", "third", "fourth", "fifth", "sixth", "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth", "thirteenth", "fourteenth", "fifteenth" };
```

Comment:
Do we really need 15? `Action<T>` and `Func<T>` top out at 8.

Reply:
lol, I just used the max `Func` would allow on .NET Standard 2.0.

Reply:
Well, I guess it doesn't hurt.